frontend/src/store/configurate-episode/actions.ts

```diff
@@ -70,4 +77,14 @@ const loadEpisode = createAsyncThunk<Episode, number, AsyncThunkConfig>(
     return episode;
   });
 
-export { create, edit, loadEpisode };
+const deleteEpisode = createAsyncThunk<void, DeleteActionEpisodePayload, AsyncThunkConfig>(
```

Comment:
```suggestion
const delete = createAsyncThunk<void, DeleteActionEpisodePayload, AsyncThunkConfig>(
```

Reply:
Can`t rename, becouse "delete" is name of native js function. ('delete' is not allowed as a variable declaration name.ts(1389))

Reply:
Good! 👍